addons/binding/org.openhab.binding.zwave/ESH-INF/thing/fibaro_fgbs001_0_0.xml

```diff
@@ -73,6 +73,8 @@
       <property name="model">FGBS001</property>
       <property name="manufacturerId">010F</property>
       <property name="manufacturerRef">0501:0102,0501:1002,0501:3002</property>
+      <property name="commandClass:SENSOR_ALARM:1"></property>
```

Comment:
both entries are missing supportedGetSupported=false. I send you a PM with a request if you could add this to the database but I am not sure if you received it ....